test/testenv/rancher.go

```diff
@@ -162,7 +162,7 @@ func DeployRancher(ctx context.Context, input DeployRancherInput) {
 		"replicas":                  "1",
 	}
 	if input.RancherFeatures != "" {
-		values["features"] = input.RancherFeatures
+		values["CATTLE_FEATURES"] = input.RancherFeatures
```

Comment:
This is interesting. When i disable via helm on the command line i use:

```
➜ helm install rancher rancher-latest/rancher --namespace cattle-system --create-namespace --set bootstrapPassword=admin --set replicas=1  --set hostname=tunnel.fruitcase.dev --set global.cattle.psp.enabled=false --version 2.7.7 \
    --set 'extraEnv[0].name=CATTLE_FEATURES' \
    --set 'extraEnv[0].value=embedded-cluster-api=false'
```

Reply:
Your snippet is exactly what is used in https://github.com/rancher-sandbox/rancher-turtles/blob/8f7c5dc2546793ac5d8213056592b6766d497e75/scripts/turtles-dev.sh#L53 which looks equivalent to `--set CATTLE_FEATURES=embedded-cluster-api=false`.